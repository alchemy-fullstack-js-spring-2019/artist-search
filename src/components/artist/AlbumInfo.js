import React from 'react';
import PropTypes from 'prop-types';

function AlbumInfo({ year, title }) {
  return (
    <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
      <h3>{title}</h3>
      <h4>{year}</h4>
    </div>
  );
}

AlbumInfo.propTypes = {
  title: PropTypes.string.isRequired,
  year: PropTypes.string.isRequired
};

export default AlbumInfo;


