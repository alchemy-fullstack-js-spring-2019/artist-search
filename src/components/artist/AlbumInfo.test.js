import React from 'react';
import { shallow } from 'enzyme';
import AlbumInfo from './AlbumInfo';

describe('AlbumInfo component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<AlbumInfo year='' title='' />);
    expect(wrapper).toMatchSnapshot();
  });
});
