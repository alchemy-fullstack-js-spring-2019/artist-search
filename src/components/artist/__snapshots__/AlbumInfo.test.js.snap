// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlbumInfo component matches snapshot 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "column",
    }
  }
>
  <h3 />
  <h4 />
</div>
`;
